src/main.ts

```diff
@@ -73,6 +74,21 @@ class LocalTableland {
     // TODO: I don't think this is doing anything anymore...
     this.#_cleanup();
 
+    // check if the port is in use and retry 5 times (1 second between each)
```

Comment:
i figured no need for something like exponential backoff...simple probing every 1 second. can adjust if needed.